ggvenn.Rproj

```diff
@@ -0,0 +1,17 @@
+Version: 1.0
```

Comment:
I am OK to add Rproj files, although I am not using Rstudio very often yet. I am using vim, which has slightly different indent rules. Therefore, I accept the indent changes.